### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MiDiu98**

### CODE REVIEW

**Comment by MiDiu98** [comment]
# [Codecov](https://codecov.io/gh/MiDiu98/BookManagement/pull/12?src=pr&el=h1) Report
> :exclamation: No coverage uploaded for pull request base (`master@32e9d10`). [Click here to learn what that means](https://docs.codecov.io/docs/error-reference#section-missing-base-commit).
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/MiDiu98/BookManagement/pull/12?src=pr&el=tree)

```diff
@@            Coverage Diff            @@
##             master      #12   +/-   ##
=========================================
  Coverage          ?   25.71%           
  Complexity        ?       63           
=========================================
  Files             ?       35           
  Lines             ?      420           
  Branches          ?       21           
=========================================
  Hits              ?      108           
  Misses            ?      305           
  Partials          ?        7           
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/MiDiu98/BookManagement/pull/12?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/MiDiu98/BookManagement/pull/12?src=pr&el=footer). Last update [32e9d10...6873271](https://codecov.io/gh/MiDiu98/BookManagement/pull/12?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

File: README.md [hunk 1]
```diff
@@ -2,3 +2,38 @@
...
+It should contain five pages, namely Login, Register, Books List (search, order, pagination), Book Details, Book Creation and Book Edition.
+```
+
+## Features
+#### Authentication
+- Register
+- Login
+#### Main Function
+- Show all users
+- Show all enable books with pagination
```
Comment (thanglenova) [inline README.md:19]: Show disabled books chứ e?

File: README.md [hunk 2]
```diff
@@ -2,3 +2,38 @@
...
+- Show detail book
+- Search book by title and author
+- Update, delete my account
+- Create, update, delete my books
+- Create, update, delete my comments
+
+## Prerequisites
+- Java
+- Gradle
+- PostGreSQL
```
Comment (thanglenova) [inline README.md:32]: Hướng dẫn kết nối database lun e nhé!

File: README.md (hunk 1 above)
Comment (MiDiu98) [reply README.md:19]: Fixed

File: README.md (hunk 2 above)
Comment (MiDiu98) [reply README.md:33]: Fixed

File: src/main/java/com/ungmydieu/BookManagement/models/dto/CommentDTO.java [hunk 3]
```diff
@@ -19,6 +19,8 @@
     @NotBlank
     private String message;
 
+    private String author;
+
```
Comment (thanglenova) [inline src/main/java/com/ungmydieu/BookManagement/models/dto/CommentDTO.java:23]: field này làm gì vậy e?

File: README.md [hunk 4]
```diff
@@ -2,3 +2,66 @@
...
+- Create database 
+
+### Connect to PostgreSQL from Spring
+-  Change file application.properties
+```
+## Spring DATASOURCE (DataSourceAutoConfiguration & DataSourceProperties)
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.url=jdbc:postgresql://localhost:port/database
+spring.datasource.username=yourUsername
+spring.datasource.password=yourPassword
```
Comment (thanglenova) [inline README.md:47]: Chỉ cần show 2 config này được rồi e

File: README.md (hunk 1 above)
Comment (thanglenova) [reply README.md:19]: Show list disabled books e

File: src/main/java/com/ungmydieu/BookManagement/models/dto/CommentDTO.java (hunk 3 above)
Comment (MiDiu98) [reply src/main/java/com/ungmydieu/BookManagement/models/dto/CommentDTO.java:23]: Author em lưu tên user comment ạ, author = firstname + lastname, để hiển thị user comment, có field userId kèm theo nữa ạ

File: README.md (hunk 4 above)
Comment (MiDiu98) [reply README.md:45]: Fixed

File: src/main/java/com/ungmydieu/BookManagement/models/dto/CommentDTO.java (hunk 3 above)
Comment (thanglenova) [reply src/main/java/com/ungmydieu/BookManagement/models/dto/CommentDTO.java:23]: vậy thì dùng fullname đi e

File: README.md (hunk 1 above)
Comment (MiDiu98) [reply README.md]: Fixed

File: src/main/java/com/ungmydieu/BookManagement/models/dto/CommentDTO.java (hunk 3 above)
Comment (MiDiu98) [reply src/main/java/com/ungmydieu/BookManagement/models/dto/CommentDTO.java]: Fixed

**Pull request merged**